brain/python/helpers.h

```diff
@@ -57,6 +59,15 @@ inline boost::python::list toPythonList( const std::vector< T >& vector )
     return result;
 }
 
+inline uint32_ts toVector( const GIDSet& gids )
```

Comment:
but you can still implement toVector() with the above mentioned constructor, no?